Servlets/src/main/java/ua/goit/servlets/dao/CategoriesList.java

```diff
@@ -0,0 +1,27 @@
+package ua.goit.servlets.dao;
+
+import java.util.ArrayList;
+import java.util.List;
+import ua.goit.servlets.model.Category;
+
+public class CategoriesList {
+
+	public static List<Category> newCategoryInstance() {
```

Comment:
The name is not consistent with what it actually does. It returns list of all categories so should be something like `getAllCategories`